app/test_split_integer.py

```diff
@@ -2,20 +2,23 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(32, 6)) == 32
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert split_integer(32, 2) == [16, 16]
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert split_integer(32, 1) == [32]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    assert split_integer(32, 6) == [5, 5, 5, 5, 6, 6]
```

Comment:
The test `test_parts_should_be_sorted_when_they_are_not_equal` assumes a specific distribution of parts. Ensure that the `split_integer` function's logic aligns with this expectation, or adjust the test to match the actual output of the function.